pkg/ceph/mon/agent.go

```diff
@@ -207,3 +214,18 @@ func writeBackendFile(monDataDir, backend string) error {
 	}
 	return nil
 }
+
+func generateMonMap(context *clusterd.Context, cluster *ClusterInfo, folder string) (string, error) {
+	path := path.Join(folder, "monmap")
+	args := []string{path, "--create", "--clobber", "--fsid", cluster.FSID}
```

Comment:
If for some reason mkfs needs to run again (because of failure the first time), we need to make sure the monmap is the latest from our authoritative source. During development I hit it initially when debugging, but since that issue is fixed I'm not anticipating we would expect to need it in the wild.